package.json

```diff
@@ -33,13 +33,9 @@
         "cy:run": "d2-utils-cypress run --appStart 'yarn cy:server' --waitOn 'http-get://localhost:5001'",
         "demo": "EXTEND_ESLINT=true build-storybook -o ./dist/demo --quiet -s .storybook/static",
         "docs": "d2-utils-docsite build ./docs -o ./dist --jsdoc ./packages/**/src --jsdoc-output-file api.md",
-        "format": "yarn format:js && yarn format:text",
-        "format:staged": "yarn format:js --staged && yarn format:text --staged",
-        "format:js": "d2-style js apply",
-        "format:text": "d2-style text apply",
-        "lint": "concurrently -n w: \"yarn:lint:*\"",
-        "lint:js": "d2-style js check",
-        "lint:text": "d2-style text check",
+        "format": "d2-style apply",
+        "format:staged": "yarn apply --staged",
```

Comment:
I suspect the `yarn` here is supposed to be a `d2-style`.

Reply:
well spotted, fixed now.